src/cookbook/serverless-blog.md

```diff
@@ -0,0 +1,349 @@
+
+## Install
+
+Run this in your command line:
+
+```bash
+npm install buttercms --save
+```
+
+Butter can also be loaded using a CDN:
+
+```html
+<script src="https://cdnjs.buttercms.com/buttercms-1.2.7.min.js"></script>
+```
+
+## Base Example
+
+::: info
+You can find your api token on the [settings page](https://buttercms.com/settings/)
+:::
+
+Create file `buttercms.js` to create your butter instance using your API token:
+
+```javascript
+var butter = require('buttercms')('your_api_token');
+module.exports = butter
+```
+
+Using ES6:
+
+```javascript
+import Butter from 'buttercms';
+export const butter = Butter('your_api_token');
+```
+
+You could also create your butter instance using CDN:
+
+```html
+<script src="https://cdnjs.buttercms.com/buttercms-1.2.7.min.js"></script>
+<script>
+  var butter = Butter('your_api_token');
+</script>
+```
+
+Import this file into any component you want to use ButterCMS and add this code snippet to `onMounted`:
+
+```javascript
+butter.post.list({page: 1, page_size: 10}).then(function(response) {
+  console.log(response)
+})
+```
+
+This API request fetches your blog posts. Your account comes with one example post which you'll see in the response.
+
+## Display all posts and post details
+
+To display posts we create a `/blog` route (using Vue Router) in our app and fetch blog posts from the Butter API, as well as a `/blog/:slug` route to handle individual posts.
+
+Update routes in `router/index.js`:
+
+```javascript
+// ...
+import BlogHome from "@/views/BlogHome.vue"
+import BlogPost from "@/views/BlogPost.vue"
+
+const routes = [
+    // ...
+    {
+        component: BlogHome,
+        name: "Blog",
+        path: "/posts",
+    },
+    {
+        component: BlogPost,
+        name: "BlogPost",
+        path: "/posts/:slug",
+    }
+]
+
+// ...
+```
+
+Then create `views/BlogHome.vue` which will be your blog homepage that lists your most recent posts:
+
+```html
+<template>
+    <div id="blog-home">
+        <h1>Blog</h1>
+        <!-- Create `v-for` and apply a `key` for Vue. -->
+        <div
+            v-for="post in posts"
+            :key="post.slug"
+        >
+        <router-link :to="{ name: 'BlogPost', params: { slug: post.slug } }">
+            <article class="media">
+            <figure>
+                <!-- Bind results using a `:` -->
+                <!-- Use a `v-if`/`else` if there is a `featured_image` -->
+                <img
+                    v-if="post.featured_image"
+                    :src="post.featured_image"
+                    alt=""
+                >
+                <img
+                    v-else
+                    src="http://via.placeholder.com/250x250"
+                    alt=""
+                >
+            </figure>
+            <h2>{{ post.title }}</h2>
+            <p>{{ post.summary }}</p>
+            </article>
+        </router-link>
+        </div>
+    </div>
+</template>
+
+<script>
+import { butter } from "@/buttercms"
```

Comment:
lets move this down below `import { RouterLink }` as we are third-party and want to ensure that Vue is the prime import.